go.mod

```diff
@@ -1,8 +1,13 @@
 module github.com/algorand/msgp
 
-go 1.16
+go 1.17
 
 require (
 	github.com/ttacon/chalk v0.0.0-20160626202418-22c06c80ed31
 	golang.org/x/tools v0.0.0-20200423205358-59e73619c742
 )
+
+require (
+	golang.org/x/mod v0.2.0 // indirect
+	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
```

Comment:
Why is this in a separate require block? did go mod tidy create it this way?

Reply:
Yes, it was from running tidy. I left it that way since I liked the fact that direct deps were being separated from indirect. Not aware of whether this is a convention or something we don't do in other repos.